src/components/InputField/styles/InputField.ts

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components'
+
+export const InputFieldStyled = styled.div`
+  width: 100%;
+`
+
+export const InputFieldTitle = styled.p.attrs({ colour: 'white' })`
+  margin: 0px;
+
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 50px;
+
+  color: ${(props) => props.colour};
+`
+
+export const TextInput = styled.input.attrs({ borderColorInactive: 'gray', borderColor: 'white' })`
+  width: 100%;
+
+  outline: 0;
+  border: 0;
+  border-bottom: 1px solid ${(props) => props.borderColorInactive};
+
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 50px;
```

Comment:
this too, if its intentional then seems ok but might be a little odd,. but do consider making font size responsive using `clamp()`

Reply:
fixed in 8a9e69fb99032cf3ebc79e5502d5b6a086a45f1e, organised font params in theme file